app/assets/javascripts/relationships.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Remove unused file

Reply:
@mdmahmudurrahman thank you I just have deleted.